js-project-guidelines.md

```diff
@@ -336,6 +337,10 @@ We suggest either of these:
 ]
 ```
 
+##### Building
+
+You can get a pre-bundled version by running `npm run build`, an npm script we add to the `package.json`. This calls `aegir-build`, which itself calls `gulp build`. This is available for every project that uses `aegir`.
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1211152/24947580/43b30880-1f5f-11e7-897e-7db1060b4cc8.png)

just realised now that I never submitted this CR :(